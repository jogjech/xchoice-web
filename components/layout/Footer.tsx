import React, { FunctionComponent } from "react";
import { Layout } from "antd";
import styles from "./footer.module.css";

interface Props {}

const Footer: FunctionComponent<Props> = () => {
  return <Layout.Footer id={styles.siteFooter}>Kevin Wang Â©2020</Layout.Footer>;
};

export default Footer;
