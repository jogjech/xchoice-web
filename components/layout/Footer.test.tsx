import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Footer from "./Footer";

describe("Page footer", () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer></Footer>);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
