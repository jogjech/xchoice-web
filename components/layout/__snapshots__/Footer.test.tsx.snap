// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page footer renders correctly 1`] = `
<Footer
  id="siteFooter"
>
  Kevin Wang Â©2020
</Footer>
`;
